internal/controller/authenticator.go

```diff
@@ -162,6 +165,14 @@ func SetUpOAuthConnection(
 	}
 	tokenSource := GetTokenSource(contextWithTimeout, oauthConfig, tlsConfig)
 
+	// Log proxy settings
+	proxyEnv := os.Getenv("HTTPS_PROXY")
+	if proxyEnv != "" {
+		logger.Info("[Proxy Set] HTTPS_PROXY is set", zap.String("proxy", proxyEnv))
+	} else {
+		logger.Info("[Proxy Unset] HTTPS_PROXY is not set")
+	}
```

Comment:
1. Remove the fancy formatting and keep it simple. When you want to add structure to log entries, use fields.
1. `http.ProxyFromEnvironment`, which you use below, is documented to use multiple environment variables. Log all of them.
```suggestion
	logger.Info(
		"Configuring proxy from environment variables",
		zap.String("http_proxy", os.Getenv("HTTP_PROXY")),
		zap.String("https_proxy", os.Getenv("HTTPS_PROXY")),
		zap.String("no_proxy", os.Getenv("NO_PROXY")),
	)
```